src/server/README.md

```diff
@@ -549,6 +548,61 @@ Student API
             -   errors:
                 -   Missing fields in body -> Status code: 400
                 -   Internal server error -> Status code: 500
+                
+    -   api/student/get-applied-job
+    
+        -   Returns a list of JobApplication objects associated with the user.
+        -   Body: None
+        -   Authorization restrictions:
+            -   User must be logged in
+            -   User must be a student
+        -   Parameters: None
+        -   Response:
+            -   success:
+                Status code: 200
+                ```
+                {
+                    jobs: [
+                        {
+                            id: number,
+                            date: Date,
+                            job: {
+                                id: number,
+                                targetYears: string[],
+                                hoursPerWeek: number,
+                                description: string,
+                                expirationDate: "Date,
+                                startDate: Date,
+                                endDate: Date,
+                                postedOn: Date,
+                                type: string[],
+                                title: string,
+                                status: 'Hiring' | 'Closed',
+                                minSalary: number,
+                                maxSalary: number,
+                                departmentId: string,
```

Comment:
We need to change the type of departmentId from string to a departmentId object. Maybe in the next PR.